From e1144dc7a097fdc61533eb9ad7dcc7cda51a8052 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Thu, 13 Apr 2023 15:10:03 +0200
Subject: [PATCH] configs: bytedevkit: add SPI drivers, use default prompt

Support different SPI NOR manufacturers due to assembly change.
Use default prompt to simplify production test.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
---
 configs/stm32mp157_bytedevkit_1g_defconfig | 7 ++++++-
 configs/stm32mp157_bytedevkit_defconfig    | 7 ++++++-
 2 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/configs/stm32mp157_bytedevkit_1g_defconfig b/configs/stm32mp157_bytedevkit_1g_defconfig
index 33352016e2..07b538cbec 100644
--- a/configs/stm32mp157_bytedevkit_1g_defconfig
+++ b/configs/stm32mp157_bytedevkit_1g_defconfig
@@ -21,7 +21,6 @@ CONFIG_SPL_POWER_SUPPORT=y
 CONFIG_SYS_SPI_U_BOOT_OFFS=0x80000
 CONFIG_SPL_WATCHDOG_SUPPORT=y
 CONFIG_HUSH_PARSER=y
-CONFIG_SYS_PROMPT="byteDEVKIT> "
 CONFIG_CMD_CPUINFO=y
 # CONFIG_CMD_BOOTD is not set
 CONFIG_CMD_BOOTZ=y
@@ -83,6 +82,12 @@ CONFIG_MTD=y
 CONFIG_DM_MTD=y
 CONFIG_SYS_MTDPARTS_RUNTIME=y
 CONFIG_DM_SPI_FLASH=y
+CONFIG_SPI_FLASH_ATMEL=y
+CONFIG_SPI_FLASH_ISSI=y
+CONFIG_SPI_FLASH_MACRONIX=y
+CONFIG_SPI_FLASH_SPANSION=y
+CONFIG_SPI_FLASH_STMICRO=y
+CONFIG_SPI_FLASH_SST=y
 CONFIG_SPI_FLASH_WINBOND=y
 # CONFIG_SPI_FLASH_USE_4K_SECTORS is not set
 CONFIG_SPI_FLASH_MTD=y
diff --git a/configs/stm32mp157_bytedevkit_defconfig b/configs/stm32mp157_bytedevkit_defconfig
index 78a880e4e6..7472b8c6c1 100644
--- a/configs/stm32mp157_bytedevkit_defconfig
+++ b/configs/stm32mp157_bytedevkit_defconfig
@@ -20,7 +20,6 @@ CONFIG_SPL_POWER_SUPPORT=y
 CONFIG_SYS_SPI_U_BOOT_OFFS=0x80000
 CONFIG_SPL_WATCHDOG_SUPPORT=y
 CONFIG_HUSH_PARSER=y
-CONFIG_SYS_PROMPT="byteDEVKIT> "
 CONFIG_CMD_CPUINFO=y
 # CONFIG_CMD_BOOTD is not set
 CONFIG_CMD_BOOTZ=y
@@ -82,6 +81,12 @@ CONFIG_MTD=y
 CONFIG_DM_MTD=y
 CONFIG_SYS_MTDPARTS_RUNTIME=y
 CONFIG_DM_SPI_FLASH=y
+CONFIG_SPI_FLASH_ATMEL=y
+CONFIG_SPI_FLASH_ISSI=y
+CONFIG_SPI_FLASH_MACRONIX=y
+CONFIG_SPI_FLASH_SPANSION=y
+CONFIG_SPI_FLASH_STMICRO=y
+CONFIG_SPI_FLASH_SST=y
 CONFIG_SPI_FLASH_WINBOND=y
 # CONFIG_SPI_FLASH_USE_4K_SECTORS is not set
 CONFIG_SPI_FLASH_MTD=y
-- 
2.30.2

